### ISSUE TITLE
Visual Studio for Mac

### ISSUE THREAD

**Issue by JamesItSolutionsLtd** [MicrosoftDocs/visualstudio-docs#1151]
What about an example of how to get this to work on visual studio for mac

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ca46de41-5103-a318-3553-3bf05c59751b
* Version Independent ID: 27b60762-dfe8-9320-c891-164e2f0ca980
* Content: [Code Generation in a Build Process in Visual Studio - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/modeling/code-generation-in-a-build-process)
* Content Source: [docs/modeling/code-generation-in-a-build-process.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/modeling/code-generation-in-a-build-process.md)
* Product: **visual-studio-dev15**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**

**Comment by gewarren** [comment]
@asb3993 Amy, any idea if T4 text transformation is supported on VS for Mac?

**Comment by asb3993** [comment]
Yep, it's supported in Visual Studio for Mac. There are actually t4 templates built in, you can find these by right-clicking on your project and selecting "Add > New File". There are three templates under the "text templating" section.
